packager/signer/signer.go

```diff
@@ -495,6 +495,12 @@ func (this *Signer) serveSignedExchange(resp http.ResponseWriter, fetchResp *htt
 		duration = maxAge
 	}
 	date := now.Add(-24 * time.Hour)
+	expires := date.Add(duration)
+	if !expires.After(now) {
+		log.Printf("Not packaging because computed max-age %d places expiry in the past\n", metadata.MaxAgeSecs)
+		proxy(resp, fetchResp, fetchBody)
+		return
+	}
 	signer := signedexchange.Signer{
 		Date:        date,
 		Expires:     date.Add(duration),
```

Comment:
Should you use "expires" (from above) here?

Reply:
Oops, yes.